src/Shared/Game/PremiumState.cs

```diff
@@ -26,7 +26,7 @@ namespace Melia.Shared.Game
 		/// <summary>
 		/// Returns whether the premium state is active.
 		/// </summary>
-		public bool Active => this.Expiration > DateTime.Now;
+		public bool Active => Type == PremiumType.Token && this.Expiration > DateTime.Now;
```

Comment:
Hmm, maybe there should instead be a "TokenActive" method?  Because we need to know if you actually have Token status and not something else.  Like I suppose you could just check Active and PremiumType = Token but that feels clunky to me.